ecosystem/indexer-grpc/indexer-grpc-table-info/src/internal_indexer_db_service.rs

```diff
@@ -166,31 +166,32 @@ impl InternalIndexerDBService {
 
     pub async fn run(&mut self, node_config: &NodeConfig) -> Result<()> {
         let mut start_version = self.get_start_version(node_config).await?;
+        let mut target_version = self.db_indexer.main_db_reader.ensure_synced_version()?;
```

Comment:
this will error out with an empty db (before genesis is put in), is that intentional?

Reply:
yes, internal indexer is supposed to start after main db bootstrapped.

Reply:
okay..